services/scan/schema.sql

```diff
@@ -1,3 +1,18 @@
+create table election (
+  -- enforce singleton table
```

Comment:
Do we actually want to enforce this? If so, then I guess I feel less certain about my foreign key comment from earlier.

Reply:
I think that we do. Right now the product expectation is that all devices but VxAdmin are only ever configured for one election at a time. Might as well enforce that expectation and get the benefit of never accidentally creating two elections. Easy to remove later if necessary.

And yes, this is why I was ambivalent about the foreign keys. I will leave them off for now.